Bottom: da4a3cc484eca962d23e6a6deba3e4baf43341ed
Top:    f863b2f605bc31512b42dcd352b343cfdd616053
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:43:45 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
